var/spack/repos/builtin/packages/py-spglib/package.py

```diff
@@ -10,9 +10,11 @@ class PySpglib(PythonPackage):
     """Python bindings for C library for finding and handling
     crystal symmetries."""
 
-    homepage = "https://atztogo.github.io/spglib/"
+    homepage = "http://spglib.github.io/spglib"
     pypi = "spglib/spglib-1.9.9.18.tar.gz"
+    git = "https://github.com/spglib/spglib.git"
 
+    version("2.0.2", sha256="1d081ec22da4ab4fc3198e9445ddad6dec2261c43927831151d93e39422610aa")
     version("1.16.1", sha256="9fd2fefbd83993b135877a69c498d8ddcf20a9980562b65b800cfb4cdadad003")
     version("1.9.9.18", sha256="cbbb8383320b500dc6100b83d5e914a26a97ef8fc97c82d8921b10220e4126cd")
 
```

Comment:
Is setuptools actually used at run-time?

The comment below suggests that I added this. However, none of the versions of this package have ever imported setuptools/pkg_resources at run-time. If my memory serves me correctly, I added this comment to several packages where the source code didn't import, but the package had an entrypoint that auto-generated a file in `prefix.bin` that imported it. I have a feeling that this feature was only for older versions of Python/pip/setuptools/something but I don't know when it changed. I'm personally fine with removing it until someone complains.

Reply:
You are right it is not needed anymore. I checked with the example from [readthedocs](https://spglib.readthedocs.io/en/latest/python-spglib.html) (bottom) and it worked fine without it.